From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 11 Mar 2022 12:44:46 -0500
Subject: [PATCH] pesign: print digests before filenames like sha256sum does

Most digest tools print the digest before the filename, there's no
reason pesign needs to be different.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/file_pe.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
